nflow-jetty/src/main/java/io/nflow/jetty/mapper/StateVariableValueTooLongExceptionMapper.java

```diff
@@ -1,18 +0,0 @@
-package io.nflow.jetty.mapper;
-
-import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
-
-import javax.ws.rs.core.Response;
-import javax.ws.rs.ext.ExceptionMapper;
-import javax.ws.rs.ext.Provider;
-
-import io.nflow.engine.workflow.executor.StateVariableValueTooLongException;
-import io.nflow.rest.v1.msg.ErrorResponse;
-
-@Provider
-public class StateVariableValueTooLongExceptionMapper implements ExceptionMapper<StateVariableValueTooLongException> {
```

Comment:
I think there is a tradeoff between putting the error catching&converting logic to one place vs replicating it in every method of the api.
I think we could have one generic exception mapper that just catches Throwable and returns ErrorResponse object.